pkg/ingester/wal.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+package ingester;
+
+option go_package = "ingester";
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+import "github.com/cortexproject/cortex/pkg/ingester/client/cortex.proto";
+
+message Record {
+  string user_id = 1;
+  repeated Labels labels = 2 [(gogoproto.nullable) = false];
```

Comment:
Are these ever set? If not they should be removed from the proto, along with `message Labels`

Reply:
Yes, they are set here https://github.com/cortexproject/cortex/pull/1103/files#diff-a13abfbfe345455f52de41855f82b320R232 to indicate new series created.